cmd/platform/main.go

```diff
@@ -154,7 +154,7 @@ func main() {
 	}
 
 	publisher := gateways.NewPublisher(storage, identityService, claimsService, mtService, keyStore, transactionService, proofService, publisherGateway, networkResolver, ps)
-	packageManager, err := protocol.InitPackageManager(ctx, networkResolver.GetSupportedContracts(), cfg.Circuit.Path)
+	packageManager, err := protocol.InitPackageManager(ctx, networkResolver.GetSupportedContracts(), cfg.Circuit.Path, auth.UniversalDIDResolver)
```

Comment:
it is better to provide an ability to configure and pass it both to auth verifier and here, so production will not rely on dev instance of third party project. (as fallback it is ok)

Reply:
solved